paasta_tools/adhoc_tools.py

```diff
@@ -95,7 +95,7 @@ def __init__(
         )
 
     def __repr__(self) -> str:
-        return "AdhocJobConfig(%r, %r, %r, %r, %r, %r)" % (
+        return "AdhocJobConfig({!r}, {!r}, {!r}, {!r}, {!r}, {!r})".format(
```

Comment:
why is this one using .format instead of an f string?

Reply:
see the note here: https://github.com/asottile/pyupgrade#f-strings